bin/util.js

```diff
@@ -221,16 +180,14 @@ exports.verify = async function(program) {
  * 4. decode the actual `signature` paramter to bytes (not base 64)
  * 5. pass publicKey, canonziedString, and decoded signature bytes to verify
  */
-  const requestOptions = await getHTTPMessage();
   const {
-    headers = '', keyType, publicKey,
+    headers = '', keyType,
```

Comment:
unusual style going on here.  Usually if we break like this it's one param per line, but that's usually if it doesn't fit on one line.  So, two params then a break is unconventional.

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/c3db9340863113950472faa881437a6dcc154eaa